app/components/recirculation-prefooter.js

```diff
@@ -58,17 +58,7 @@ export default Component.extend(
 				listRendered: defer()
 			});
 
-			if (!this.get('fastboot.isFastBoot')) {
```

Comment:
The removed part is crucial for switching PreFooter with BLB. Are you sure you need to delete it?